AmahiAnywhere/AmahiAnywhere/Presentation/Login/LoginViewController.swift

```diff
@@ -29,6 +29,11 @@ class LoginViewController: BaseUIViewController {
         (textField as! SkyFloatingLabelTextField).errorMessage = nil
         return true
     }
+    
+    @IBAction func userClickForgotPassword(_ sender: UIButton) {
+        UIApplication.shared.open(NSURL(string:"https://www.amahi.org/forgot")! as URL)
```

Comment:
It would have been better if you have used `UR`' instead of `NSURL`.
Also, it's never recommended to hardcode the URL, you should define base URL and URL scheme for the app.

Also, force unwrapping while defining is not good (it can crash the app), you should use `if let` or `guard let` statements instead.